base_zambezi_pdx235_defconfig

```diff
@@ -1,11 +1,4 @@
 CONFIG_ARCH_SONY_ZAMBEZI=y
 CONFIG_MACH_SONY_PDX235=y
-# CONFIG_NFC_QTI_I2C is not set
-# CONFIG_FP_ET613 is not set
-CONFIG_FP_EC617=y
-CONFIG_TOUCHSCREEN_FTS=y
-CONFIG_TOUCHSCREEN_FTS_DIRECTORY="focaltech_touch"
-# CONFIG_TOUCHSCREEN_SEC_TS is not set
-CONFIG_SMB1398_CHARGER=y
-CONFIG_SMB1355_SLAVE_CHARGER=y
-CONFIG_LEDS_AW2016=y
```

Comment:
please don't remove the config for the missing drivers

Reply:
@jerpelea Zambezi uses QG+SMB5 combo, other entries either have alternative names (e.g. stock: CONFIG_FP_EC617 -> sodp: CONFIG_SENSORS_ET51X) or not used at all (CONFIG_LEDS_AW2016, PDX235 has no LED). they were not blindly removed